src/orquestra/sdk/mlflow/_connection_utils.py

```diff
@@ -33,18 +33,6 @@ def _is_executing_remoteley() -> bool:
     """
     if os.getenv(_env.CURRENT_CLUSTER_ENV):
         return True
-    else:
-        # TODO: https://zapatacomputing.atlassian.net/browse/ORQSDK-914 - this
-        # workaround exists to maintian compatibility with previous WDR versions.
-        # Future releases should remove this else block.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏This is only a breaking change if a cluster is on a version from before July 21